deltacat/utils/pyarrow.py

```diff
@@ -569,8 +569,8 @@ def s3_file_to_table(
                 **s3_client_kwargs,
             )
 
-        if READER_TYPE_KWARG in kwargs:
-            kwargs.pop(READER_TYPE_KWARG)
+    if READER_TYPE_KWARG in kwargs:
+        kwargs.pop(READER_TYPE_KWARG, None)
```

Comment:
The second argument is redundant.

Reply:
If the kwarg isn't present for some reason, it'll throw an exception, so I just wanted to avoid that unnecessary error

Reply:
Oh whoops brain fart, yeah I can just remove it due to the if statement 💀

Reply:
:)